Cargo.toml

```diff
@@ -150,6 +150,37 @@ default-members = [
     "ecosystem/node-checker",
 ]
 
```

Comment:
Can we please add rust-version to avoid confusion after the upgrade as it will break compile in weird ways for anyone on an older version (e.g., on their laptop; learned the hard way trying to compile faucet)
https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field

My understanding is we literally just have to set
```
[package]
rust-version = "1.56"
````

Reply:
> Can we please add rust-version to avoid confusion after the upgrade as it will break compile in weird ways for anyone on an older version (e.g., on their laptop; learned the hard way trying to compile faucet) https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field
> 
> My understanding is we literally just have to set
> 
> ```
> [package]
> rust-version = "1.64"
> ```

Done. One needs to use the new tag `[workspace.package]` for this. We can put all other info we want to inherit to workspace crates there as well, but did not do this for now.